### PR TITLE
bullet pouch

### PR DESCRIPTION
**Pull request by bandti45**
**Summary**
SUMMARY: Content "Addition of an ammo bag"

**Purpose of change**
To address half of issue #32230 this PR adds a second 'leather pouch' that will hold small ammunition the same way a quiver does 

**Describe the solution**
Copy the recipe and item of leather pouch, then remove the storage capacity and add the bandolier quality with the addition of every small (low volume) ammunition to its accepted list

**Additional context**
The other half of the issue needs changes to existing equipment to be

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -5809,6 +5809,78 @@
     "material_thickness": 1,
     "flags": [ "WAIST", "OVERSIZE", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "ammo_pouch",
+    "type": "ARMOR",
+    "name": "ammo pouch",
+    "description": "A small pouch that can be used to store most types of small ammunition, rockets wont fit.  Activate to store ammunition.",
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]:
```suggestion
    "description": "A small pouch that can be used to store most types of small ammunition, rockets won't fit.  Activate to store ammunition.",
```

File: data/json/items/armor.json [hunk 2]
```diff
@@ -5809,6 +5809,78 @@
     "material_thickness": 1,
     "flags": [ "WAIST", "OVERSIZE", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "ammo_pouch",
+    "type": "ARMOR",
+    "name": "ammo pouch",
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: Plural name needed.

**Comment by bandti45** [comment]
Thank you for that typo correction I am bad with putting those.

File: data/json/items/armor.json (hunk 2 above)
Comment (bandti45) [reply data/json/items/armor.json]: I do not agree it is a single item that its always separated from similar items. If you go into the code only armor that comes in pairs like "arm guards" have that line of text so I will not add it to this item for consistency sake.

Comment (Night-Pryanik) [reply data/json/items/armor.json]: You are wrong. Game automatically adds a trailing `s` to the item's `name` if no `name_plural` is defined. So if we don't specifically set a plural name for arm guards, we'll get `arm guardss` in the game for several items. In your case not having a defined plural name will result in `ammo pouchs` for several items. Look in the `data\json\items\armor\ammo_pouch.json`. There are a lots of pouches with a defined plural name.

Comment (bandti45) [reply data/json/items/armor.json]: So to clarify I need to set its plural to still be "ammo pouch"?

Comment (bandti45) [reply data/json/items/armor.json]: ok I see my mistake I forgot pouch needs an "es" at the end, that is my bad sorry for doubting you

File: data/json/items/armor.json [hunk 3]
```diff
@@ -5809,6 +5809,79 @@
...
+        "ampoule",
+        "300",
+        "32",
+        "762x25",
+        "flintlock",
+        "36paper",
+        "44paper",
+        "762R",
+        "paintball",
+        "crystallized_mana"
```
Comment (ZhilkinSerg) [inline data/json/items/armor.json]: Don't use item from mods (e.g. `crystallzied_mana` from `Magiclysm`) - it will cause errors when mods are not used.

Comment (bandti45) [reply data/json/items/armor.json]: ok thank you, do you know how I could add it through the mod? I am not familiar enough with the code for mods

**Pull request merged**